core/src/main/java/org/apache/iceberg/hadoop/CachingHadoopTables.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.hadoop;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.io.FileIO;
+
+public class CachingHadoopTables extends HadoopTables {
```

Comment:
It's a red flag that `HadoopTables` is used. What does caching in `FileIO` have to do with the method used to load tables ? I think  those are orthogonal concerns.

Reply:
This is now dropped. We are now testing directly against `CachingFileIO`.

Reply:
This is now dropped. We are now testing directly against `CachingFileIO`.